cmd/webhook.go

```diff
@@ -42,6 +44,30 @@ var webhookConfigCmd = &cobra.Command{
 			logrus.Fatal(err)
 		}
 
+		cert, err := cmd.Flags().GetString("cert")
+		if err == nil {
+			if len(cert) > 0 {
+				conf.Handler.Webhook.Cert = cert
+			}
+		} else {
+			logrus.Fatal(err)
+		}
+
+		tlsSkip, err := cmd.Flags().GetString("tlsskip")
+		if err == nil {
+			if len(tlsSkip) > 0 {
+				skip, err := strconv.ParseBool(tlsSkip)
+				if err != nil {
+					logrus.Fatal(err)
```

Comment:
I would add a comment, that if ParseBool fails, the value of `skip` is defaulted to `false`

Reply:
So you mean that modifying `logrus.Fatal(err)` to `conf.Handler.Webhook.TlsSkip = false`, right? I will fix it together with the other comment.

Reply:
No, the code is ok.
Please ignore this comment.